DevCycle.SDK.Server.Cloud/Api/DVCCloudClient.cs

```diff
@@ -64,6 +64,9 @@ public override IDVCApiClient GetApiClient()
             }
             catch (DVCException e)
             {
+                if (!e.IsRetryable() && (int)e.HttpStatusCode >= 400) {
```

Comment:
i ended up needing to check >=400 after all because this catches any exception including json parsing errors etc after a successful request. if the request succeeds w/ 200 but there's an exception afterward we don't want to treat that the same as a 400 level user error

Reply:
does the `!e.IsRetryable()` case return false for 500s? I assume we dont want to throw in the case of a 500

Reply:
yeah it just checks if the status code is >= 500 so it filters them out